stor/tests/test_dx.py

```diff
@@ -1177,15 +1180,16 @@ def test_posix_to_dx_fail(self):
         with pytest.raises(exceptions.NotFoundError, match='provide a valid source'):
             posix_p.copy(dx_p)
 
-    def test_posix_to_existing_dx_fail(self):
+    def test_posix_to_existing_dx(self):
         self.setup_temporary_project()
         self.setup_files(['/temp_folder/file.txt'])
         self.setup_posix_files(['/rand/file.txt'])
-        dx_p = DXPath('dx://' + self.project + ':/temp_folder')
+        dx_folder_p = DXPath('dx://' + self.project + ':/temp_folder')
         posix_p = Path('./{test_folder}/{path}'.format(
             test_folder=self.project, path='rand/file.txt'))
-        with pytest.raises(exceptions.TargetExistsError, match='will not cause duplicate file'):
-            posix_p.copy(dx_p)
+        posix_p.copy(dx_folder_p)
+        dx_p = DXPath('dx://' + self.project + ':/temp_folder/file.txt')
+        self.assertTrue(dx_p.exists())
```

Comment:
Should you also assert that the contents are from the correct file? I.e. that it actually got overwritten in DX?

Reply:
Yes, added that here and in TestUpload.